integrate-merkl/integrate-merkl-to-your-app/branding-and-integration.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
remove this page and add links to https://anglemoney.notion.site/Merkl-Media-Assets-ac2bb25ed5f146acb2ffd4ac99af8548?pvs=4